internal/pkg/cli/app_init.go

```diff
@@ -161,10 +166,15 @@ func (o *initAppOpts) Execute() error {
 }
 
 func (o *initAppOpts) createManifest() (string, error) {
-	manifest, err := manifest.CreateApp(o.AppName, o.AppType, o.DockerfilePath)
-	if err != nil {
-		return "", fmt.Errorf("generate a manifest: %w", err)
+	props := &manifest.LBFargateManifestProps{
+		AppName:    o.AppName,
+		Dockerfile: o.DockerfilePath,
+	}
+	props.Path = "/" + o.AppName
```

Comment:
Let's not even put slashes in. Let's just put the app name, and infer / and /* when we generate the rules.